package.json

```diff
@@ -156,7 +156,8 @@
       "strip-ansi": "6.0.1"
     },
     "@storybook/cli": {
-      "trim": "0.0.3"
+      "trim": "0.0.3",
+      "tar-fs": "^2.1.2"
```

Comment:
do we really need this? I think do `npm update tar-fs` should update the dep in lock and satisfies the request
We should try not to manually set this override section unless we have to